sup3r/utilities/loss_metrics.py

```diff
@@ -117,6 +117,103 @@ def __call__(self, x1, x2, sigma=1.0):
         return mmd
 
 
+class MaterialDerivativeLoss(tf.keras.losses.Loss):
+    """Loss class for the material derivative. This is the left hand side of
+    the Navier-Stokes equation and is equal to internal + external forces
+    divided by density.
+    https://en.wikipedia.org/wiki/Material_derivative
+    """
+
+    MAE_LOSS = MeanAbsoluteError()
```

Comment:
For some of these loss functions we're still assuming MAE is the go-to metric but it is not always. Consider making the class attribute simply `LOSS_METRIC` or `ERROR_METRIC`. Personally I'm 100% fine with manipulating the class attributes at runtime but the name is presumptious.